plugins/header_rewrite/header_rewrite_test.cc

```diff
@@ -336,6 +363,17 @@ test_parsing()
     END_TEST();
   }
 
+  {
+    ParserTest p(R"(set-header Alt-Svc "quic=\":443\"; v=\"35\"")");
+
+    CHECK_EQ(p.getTokens().size(), 3UL);
+    CHECK_EQ(p.getTokens()[0], "set-header");
+    CHECK_EQ(p.getTokens()[1], "Alt-Svc");
+    CHECK_EQ(p.getTokens()[2], R"(quic=":443"; v="35")")
```

Comment:
Shouldn't this line have have a ";" at the end?

Reply:
I think none of them should have it as per the definition of CHECK_EQ :-). But, we probably should change CHECK_EQ to not have the ; at the end of the define.